src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -998,7 +998,6 @@ public void MultiDexCustomMainDexFileList ([Values ("dx", "d8")] string dexTool,
 				foreach (var item in expected) {
 					Assert.IsTrue (actual.Contains (item), $"multidex.keep did not contain `{item}`");
 				}
-				Assert.AreEqual (expected.Length, actual.Length, "multidex.keep file contained more items than expected!");
```

Comment:
why does this fix the test? Just curious :)

Reply:
When r8 is doing its own thing for multidex, these two files are not in the main dex file: https://github.com/xamarin/xamarin-android/pull/3121/commits/40712ea8955bfab6a0579d14c6861440e0c79243#diff-571cf18620dfe20db6f256078c834986L978

These aren't needed for `minSdkVersion` of 19 or 21:
```
android/support/multidex/MultiDex$V4.class
android/support/multidex/MultiDex$V14.class
```
So I removed them from the list, but dx will still have them--which failed the test.